fcli-core/fcli-common/src/main/java/com/fortify/cli/common/util/FcliDataHelper.java

```diff
@@ -124,7 +124,8 @@ public static final void saveFile(Path relativePath, Object contents, boolean fa
 
     private static void writeFileWithOwnerOnlyPermissions(final Path filePath, final String contents, boolean failOnError) {
         try {
```

Comment:
Please use try-with-resources block, as otherwise an 'Unreleased Resource' vulnerability will be reported by FoD, i.e., something like:

```
try ( var fos = new FileOutputStream(...); var osw = new OutputStreamWriter(...); var writer = new BufferedWriter(...) ) {

}
```